glib/src/value.rs

```diff
@@ -369,20 +369,6 @@ pub trait ToValue {
     fn value_type(&self) -> Type;
 }
 
-// rustdoc-stripper-ignore-next
-/// Blanket implementation for all references.
-impl<T: ToValue + StaticType> ToValue for &T {
```

Comment:
I needed to remove this to auto-implement `ToValue` for `T: ValueDelegate`. Else they were in conflict.

The tests don't complain so...  🤷🏻

Reply:
I think this makes it impossible to pass `&Object` to e.g. `Object::set_property()` and only `Object` would work